apps/fabric-website-resources/src/customizations/customizations.ts

```diff
@@ -0,0 +1,158 @@
+import { createTheme, ITheme } from 'office-ui-fabric-react/lib/Styling';
+import { ICustomizations } from 'office-ui-fabric-react/lib/Utilities';
+import { IExampleCardCustomizations, IAppCustomizations } from '@uifabric/example-app-base';
+import { getNeutralVariant, getSoftVariant, getStrongVariant } from '@uifabric/variants';
+
+const defaultCustomizations: ICustomizations = {
+  settings: {
+    theme: createTheme({})
+  },
+  scopedSettings: {}
+};
+
+const wordCustomizations: ICustomizations = {
```

Comment:
These really should be in another package, but that can happen separately.

Reply:
A new package? I couldn't think of an existing package that would be a good fit. This would be something needed by both experiments and fabric-website-resources..